panacea/web/controllers/pml_controller.ex

```diff
@@ -24,14 +25,22 @@ defmodule Panacea.PmlController do
     {:error, message}
   end
 
-  defp respond({:ok, drugs}, conn) do
-    conn
-    |> put_status(:ok)
-    |> json(%{drugs: drugs})
+  defp get_ddis({:ok, drugs}) do
+    {:ok, ddis} = Panacea.Asclepius.Remote.ddis(drugs)
+    %{drugs: drugs, ddis: ddis}
+  end
+  defp get_ddis({:error, message}) do
+    {:error, message}
   end
+
   defp respond({:error, message}, conn) do
     conn
     |> put_status(:unprocessable_entity)
     |> json(%{message: message})
   end
+  defp respond(response, conn) do
```

Comment:
unsure of how pattern matching works here but perhaps an explicit `{:ok, thing}` would be better